### DISCUSSION TITLE
rathole vs wirguard performance

### DISCUSSION THREAD

**Discussion by kelke** [rathole-org/rathole#345] [Q&A]
Like many others, I am planing on hosting a service publicly without port forwarding using rathole and a VPC.
Since I have a Gigabit uplink, performance is quite important to me, as I would like to be able to saturate that link.
The alternative to rathole is a plain wireguard connection and iptables rules. Do any of you have experience regarding performance? Encrypting traffic is not actually a concern in my case.

**Comment by harryqt** [comment]
did some testing, performance is identical.. sticking to wireguard.

**Comment by typochecker** [comment]
虽然你的家用宽带带宽很大，但是VPS的带宽是很贵的，可能对性能的考虑有点多虑。
